mod_app/views.py

```diff
@@ -125,3 +132,22 @@ def get(self, request):
 
 def custom_404(request, exception=None):
     return render(request, "404.html", {}, status=404)
+
+
+def downloadAnalysis(request, pk):
+    analysis = Analysis.objects.get(pk=pk)
+    template_name = "downloads/analysis.html"
```

Comment:
I get a "TemplateDoesNotExist" error when I click the download button. Is this still being worked on?

Reply:
no...it seems it's been ignored, so will fix the gitignore